### ISSUE TITLE
[MISC_7.3.7] Praesentium velit ex est.

### ISSUE THREAD

**Issue by smockle** [smockle/action-filter-issues#1]
Ipsum vitae sed fugiat inventore qui ut. Vitae dolorem laudantium `id="submit-button"`. Et debitis ratione deleniti debitis veniam perferendis illo. Est qui aut expedita using `<a>` inside `<button>`. Maxime adipisci est iusto alias alias dolorem sit nobis. Aliquid accusamus corporis animi accusantium. Aliquam qui alias quia sint modi. Et sit accusantium eligendi est fugit occaecati.

**Fails**

[4.1.1 Parsing (level A)](https://www.w3.org/WAI/WCAG22/Understanding/parsing)

cc @smockle 

View the [repository](https://github.com/smockle/action-autolabel).

**Comment by smockle** [comment]
ℹ️ This issue exists for testing the [`smockle/action-filter-issues` action](https://github.com/smockle/action-filter-issues)
